packages/showcase/axes/custom-axes.js

```diff
@@ -35,16 +35,17 @@ const WORDS = [
   'skateboard',
   'wow',
   'such',
+  // eslint-disable-next-line react/jsx-key
   <tspan>
-    <tspan x="0" dy="1em">
+    <tspan key="" mx="0" dy="1em">
```

Comment:
We should just be able to add a `key` to the `tspan` and get rid of the `eslint-disable`.
Also, why was `x` changed to `mx` ?

Reply:
Fixed, and `mx` was a typo, thanks